### ISSUE TITLE
Breaks on Node.js 4.x

### ISSUE THREAD

**Issue by RIAstar** [dtabuenc/karma-html-reporter#29]
karma-html-reporter doesn't play nice with node 4.x
An old API is used to handle writeStream errors, which now throws an error.

Here's the stack trace:
```
TypeError: options must be a string or an object
    at new WriteStream (fs.js:1799:11)
    at Object.fs.createWriteStream (fs.js:1785:10)
    at /home/RIAstar/dev/trasys/seed/angular-app-seed/node_modules/karma-html-reporter/index.js:89:22
    at /home/RIAstar/dev/trasys/seed/angular-app-seed/node_modules/karma/lib/helper.js:82:7

```

**Comment by RIAstar** [comment]
As I was preparing a PR for this, I noticed it was [already fixed](https://github.com/RIAstar/karma-html-reporter/commit/997f091ca66dee99a7d604c8c84636ce29da86c5#diff-168726dbe96b3ce427e7fedce31bb0bc).

However this fix doesn't seem to be included in the latest release on NPM.
Any chance you'll be making a new release? It would make sense since currently the only way to make karma-html-reporter work on node 4 is to target the `master` branch instead of a stable release.

**Comment by dtabuenc** [comment]
Released 0.2.7

**Comment by RIAstar** [comment]
Cool, thanks!
